js/moveset_import.js

```diff
@@ -262,7 +260,11 @@ function addSets(pokes) {
 					currentPoke.nameProp = "Custom Set";
 				}
 				currentPoke.isCustomSet = true;
-				currentPoke.ability = getAbility(rows[i + 1].split(":"));
+				if (currentPoke.name.indexOf("-Mega") !== -1) {
```

Comment:
It's interesting that you're checking for non-mega first, and then leaving mega for the `else`. I would do it the other way around: the default should be non-mega.

And also, you should probably check that `POKEDEX_SM[currentPoke.name]` and `POKEDEX_SM[currentPoke.name].ability ` even exist.

Reply:
Honestly I just did this fast so I wouldn’t forget how I did it tomorrow lol. But yes that is probably better, ty!